taxon_names_resolver/manip_tools.py

```diff
@@ -133,7 +135,7 @@ def _slice(self, level):
         if level >= len(self.taxonomy):
             raise IndexError('Level greater than size of taxonomy')
         res = []
-        for ident in self.keys():
+        for ident in sorted(list(self.keys())):
```

Comment:
Made this sorted to return consistent ordering of the TaxonTrees for testing purposes.